client/js/main.js

```diff
@@ -0,0 +1,161 @@
+        },
+        scales: {
+            xAxes: [{
+                display: true,
+                scaleLabel: {
+                  display: true,
+                  labelString: 'Time (s)'
+                }
+              }],
+            yAxes: [{
+                display: true,
+                scaleLabel: {
+                  display: true,
+                  labelString: 'Velocity (km/h)'
+                },
+                ticks: {
+                    beginAtZero: true,
+                    max: 175
+                    }
+                }]
+            }
+        };
+var powerTimeOptions = {
+                    title:{
+                        display:true,
+                        text:'Power-Time',
+                        maintainAspectRatio: true,
+                        fontSize:14
+                    },
+
+                    legend: {
+                        display: false
+                    },
+                    scales: {
+                        xAxes: [{
+                            display: true,
+                            scaleLabel: {
+                              display: true,
+                              labelString: 'Time (s)'
+                            }
+                          }],
+                        yAxes: [{
+                            display: true,
+                            scaleLabel: {
+                              display: true,
+                              labelString: 'Power (W)'
+                            },
+                            ticks: {
+                                beginAtZero: true,
+                                max: 175
+                                }
+                            }]
+                        }
+              };
+var cadenceTimeOptions = {
+                    title:{
+                        display:true,
+                        text:'Cadence-Time',
+                        maintainAspectRatio: true,
+                        fontSize:14
+                    },
+                    
+                    legend: {
+                        display: false
+                    },
+                    scales: {
+                        xAxes: [{
+                            display: true,
+                            scaleLabel: {
+                              display: true,
+                              labelString: 'Time (s)'
+                            }
+                          }],
+                        yAxes: [{
+                            display: true,
+                            scaleLabel: {
+                              display: true,
+                              labelString: 'Cadence'
+                            },
+                            ticks: {
+                                beginAtZero: true,
+                                max: 175
+                                }
+                            }]
+                        }
+              };
+
+
+new Chart(document.getElementById("velocityTimeChart"), {
```

Comment:
Instead of the document.getElementById("..."), can you put in the variable that you set up top?